types/office-js-preview/index.d.ts

```diff
@@ -137234,7 +137522,27 @@ declare namespace PowerPoint {
              */
             $all?: boolean;
             /**
-             * Specifies the bold status of font. Returns `null` if the `TextRange` contains both bold and non-bold text fragments.
+             * Specifies whether the text in the `TextRange` is set to use the **All Caps** attribute which makes lowercase letters appear as uppercase letters. The possible values are as follows:
+             *
+             * - `true`: All the text has the **All Caps** attribute.
+             *
+             * - `false`: None of the text has the **All Caps**attribute.
+             *
+             * - `null`: Some, but not all, of the text has the **All Caps** attribute.
```

Comment:
Making the change here in the `ShapeFontLoadOptions` actually makes these descriptions even more confusing and misleading than they initially are. We need to decide as an org what we're going to do about all the `LoadOptions` interfaces. In the meantime, I recommend we leave the descriptions in this interface as you originally had them.
```suggestion
             * Specifies the bold status of font. Returns `null` if the `TextRange` contains both bold and non-bold text fragments.
```